setup.cfg

```diff
@@ -10,16 +10,23 @@ url = https://github.com/duckinator/emanate
 license = MIT
 classifiers =
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.5
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
     License :: OSI Approved :: MIT License
     Operating System :: OS Independent
 
 [options]
 include_package_data = False
 packages = find:
-python_requires = >=3.5
+
+python_requires = >=3.6
+install_requires =
+    dataclasses; python_version < '3.7'
+setup_requires =
+    dataclasses; python_version < '3.7'
```

Comment:
Since Emanate is going to be released as a zipapp, having conditional dependencies like this doesn't work. You'll need it to always be installed. (Learned that the fun way when working on Bork.)

```suggestion
# Remove `dataclasses` dependency once Python <=3.7 support is dropped.
install_requires =
    dataclasses
setup_requires =
    dataclasses
```